src/app/containers/ATIAnalytics/params/frontpage/index.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
Do tests from this file need to be moved to `src/app/containers/ATIAnalytics/params/frontpage/buildParams.test.js`?

Reply:
Yes.

Will do same for `.../article/buildParams.test.js` and `.../media/buildParams.test.js`

Reply:
Done!